runtime-linux.md

```diff
@@ -1,11 +1,5 @@
 # Linux Runtime
 
-## File descriptors
-
-By default, only the `stdin`, `stdout` and `stderr` file descriptors are kept open for the application by the runtime.
-The runtime MAY pass additional file descriptors to the application to support features such as [socket activation](http://0pointer.de/blog/projects/socket-activated-containers.html).
-Some of the file descriptors MAY be redirected to `/dev/null` even though they are open.
```

Comment:
If you're duping standard streams, it is presumably because you are about to `execve` the runtime.  Maybe you're doing that via a shell, and maybe you're doing that in code, but you're still going to have to set arguments following the command-line API.  We could keep this separate section just focused on the bundle authors (who don't have to care about how the standard streams or additional file descriptors are setup), but I think it's redundant when we have the command line API covering both the presence of these file descriptors when executing the container process *and* telling you how to set them up.  But whatever, if that's still not a convincing argument I can leave this section as it stood, just let me know.